text/0118-search-api.md

```diff
@@ -1155,6 +1184,28 @@ The beginning of a matching term within a field is indicated by `start`, and its
 
 > See [3.1.14. `showMatchesPosition`](#3116-showmatchesposition) section.
 
+###### 3.2.1.1.4. `_rankingScore`
+
+- Type: Number
+- Required: False
+
+The relevancy score of a document relative to the search query. Higher is better.
+
+`1.0` indicates a perfect match, `0.0` no match at all (Meilisearch should not return documents that don't match the query).
+
+> See [Ranking Score](./0195-ranking-score.md#31-ranking-score) for details.
+
+###### 3.2.1.1.5. `_rankingScoreDetails`
+
+- Type: Object
+- Required: False
+
+(EXPERIMENTAL) The ranking score or a document per each ranking rule and relative to the search query.
```

Comment:
(For my understanding) Why is it the ranking score or a document per ranking rule? Isn't it both? I'm sorry if I'm misunderstanding this.

Reply:
of*

typo

Reply:
fixed